### PR TITLE
docs: Fix test typo and update/correct devstack commands.

### PR DESCRIPTION
**Pull request by doctoryes**
## Description

In working on this ticket: https://openedx.atlassian.net/browse/TNL-8485 , I discovered a few places where the documentation needed an update. The needed updates are in this PR, including more details on how to run integration tests against an specific Elasticsearch instance in a Docker container.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -109,12 +109,26 @@ To save on overhead while running individual tests, from within the container, y
...
 
 .. code::
 
-   EDXAPP_RUN_BLOCKSTORE_TESTS=1 python -Wd -m pytest --ds=cms.envs.test openedx/core/lib/blockstore_api/ openedx/core/djangolib/tests/test_blockstore_cache.py openedx/core/djangoapps/content_libraries/tests/
-   EDXAPP_RUN_BLOCKSTORE_TESTS=1 python -Wd -m pytest --ds=lms.envs.test openedx/core/lib/blockstore_api/ openedx/core/djangolib/tests/test_blockstore_cache.py openedx/core/djangoapps/content_libraries/tests/
+   EDXAPP_RUN_BLOCKSTORE_TESTS=1 EDXAPP_ENABLE_ELASTICSEARCH_FOR_TESTS=1 python -Wd -m pytest --ds=cms.envs.test openedx/core/lib/blockstore_api/ openedx/core/djangolib/tests/test_blockstore_cache.py openedx/core/djangoapps/content_libraries/tests/
+   EDXAPP_RUN_BLOCKSTORE_TESTS=1 EDXAPP_ENABLE_ELASTICSEARCH_FOR_TESTS=1 python -Wd -m pytest --ds=lms.envs.test openedx/core/lib/blockstore_api/ openedx/core/djangolib/tests/test_blockstore_cache.py openedx/core/djangoapps/content_libraries/tests/
+
+To run these integration tests while using Elasticsearch, add ``EDXAPP_ENABLE_ELASTICSEARCH_FOR_TESTS=1`` on the above commands:
```
Comment (bradenmacdonald) [inline README.rst:119]: This says to `add EDXAPP_ENABLE_ELASTICSEARCH_FOR_TESTS=1`, but it's already present in the above commands. I think you meant to leave it out of the initial example commands?

Comment (doctoryes) [reply README.rst:119]: Thanks for catching that, @bradenmacdonald - fixed now.

**Review by bradenmacdonald — APPROVED** [review]
Thanks a lot, Julia! I tested these commands on my devstack to confirm they're working. The version without `EDXAPP_TEST_ELASTICSEARCH_HOST` wasn't actually working anymore but I guess that's to be expected?

**Comment by doctoryes** [comment]
Ah - right - it's because no devstack containers depend on `edx.devstack.elasticsearch` anymore - so it's not started in devstack typically when using `make dev.up.lms+studio`. Yet that address is still the default here:
https://github.com/edx/edx-platform/blob/master/cms/envs/test.py#L266
I'll create a PR to change that default.

**Comment by doctoryes** [comment]
Created a PR to change the default here:
https://github.com/edx/edx-platform/pull/28419

**Pull request merged**